third_party/webhook/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package log is originated from https://github.com/yahoo/k8s-athenz-webhook/releases/tag/v0.1.4
```

Comment:
```suggestion
// Package webhook is originated from https://github.com/yahoo/k8s-athenz-webhook/releases/tag/v0.1.4
// Copyright 2019 Verizon Media Inc. Licensed under the terms of the 3-Clause BSD License.
// See LICENSE file in github.com/yahoo/k8s-athenz-webhook/blob/v0.1.4/LICENSE for terms.
```
I think we can just add the terms in here and then remove the files in `yahoo_inc_originated_documents`.
Or, do you have other intentions for keeping the original documents too?

Reply:
@WindzCUHK If it is someone else's work, I think its a good move to preserve what they have written in an original format. Don't want to alter single word from 3rd party.

And Thank you for the suggestion!

Reply:
(And fixed typo)